blacken_docs.py

```diff
@@ -36,6 +36,13 @@
     r'(?P<after>^(?P=indent)\\end{minted}\s*$)',
     re.DOTALL | re.MULTILINE,
 )
```

Comment:
also should replace your new version instead of leaving the old `LATEX_RE` around